chains.py

```diff
@@ -59,10 +58,10 @@ def configure_llm_only_chain(llm):
     # LLM only response
     template = """
     You are a helpful assistant that helps a support agent with answering programming questions.
-    If you don't know the answer, just say that you don't know, don't try to make up an answer.
+    If you don't know the answer, just say that you don't know, you must not make up an answer.
     """
     system_message_prompt = SystemMessagePromptTemplate.from_template(template)
-    human_template = "{text}"
+    human_template = "{question}"
```

Comment:
Is the `{question}` keyword implicitly replaced with the first argument in `chain.invoke()`?

Reply:
I think this is right but it'd be good to get confirmation from @nfcampos - @jmorganca @mchiang0610 - this is one of the changes that was on the mac studio.  Do you know who should review this?